scripts/ansible/oe-windows-acc-setup.yml

```diff
@@ -8,32 +8,58 @@
   tasks:
     - name: OE setup | Set installer URLs from the OE storage account
       set_fact:
-        intel_psw_2_2_url:  "https://oejenkins.blob.core.windows.net/oejenkins/intel_sgx_win_2.2.100.47975_PV.zip"
-        intel_psw_2_3_url:  "https://oejenkins.blob.core.windows.net/oejenkins/Intel_SGX_PSW_for_Windows_v2.3.100.49777.exe"
-        intel_dcap_url:     "https://oejenkins.blob.core.windows.net/oejenkins/Intel_SGX_DCAP_for_Windows_v1.1.100.49925.exe"
-        git_url:            "https://oejenkins.blob.core.windows.net/oejenkins/Git-2.19.1-64-bit.exe"
-        seven_zip_url:      "https://oejenkins.blob.core.windows.net/oejenkins/7z1806-x64.msi"
-        vs_buildtools_2017: "https://oejenkins.blob.core.windows.net/oejenkins/vs_buildtools_2017.exe"
-        ocaml_url:          "https://oejenkins.blob.core.windows.net/oejenkins/ocpwin64-20160113-4.02.1+ocp1-mingw64.zip"
-        clang7_url:         "https://oejenkins.blob.core.windows.net/oejenkins/LLVM-7.0.1-win64.exe"
-        shellcheck_url:     "https://oejenkins.blob.core.windows.net/oejenkins/shellcheck-stable.exe"
-        nuget_url:          "https://oejenkins.blob.core.windows.net/oejenkins/nuget.exe"
-        devcon_package_url: "https://oejenkins.blob.core.windows.net/oejenkins/devcon_package.cab"
-        az_dcap_client_url: "https://oejenkins.blob.core.windows.net/oejenkins/Microsoft.Azure.DCAP.Client.1.0.0.nupkg"
+        intel_psw_2_2_url:   "https://oejenkins.blob.core.windows.net/oejenkins/intel_sgx_win_2.2.100.47975_PV.zip"
+        intel_psw_2_2_hash:  "EB479D1E029D51E48E534C284FCF5CCA3A937DA43052DCB2F4C71E5F354CA623"
+        intel_psw_2_4_url:   "https://oejenkins.blob.core.windows.net/oejenkins/Intel%20SGX%20PSW%20for%20Windows%20v2.4.100.51291.exe"
+        intel_psw_2_4_hash:  "79AE32E984B5511CE4BF7568403333F837FBCE7E8D5730271C5D68F55BBF251D"
+        intel_dcap_url:      "https://oejenkins.blob.core.windows.net/oejenkins/Intel%20SGX%20DCAP%20for%20Windows%20v1.2.100.49925.exe"
+        intel_dcap_hash:     "F31E4451CA32E19CA3DCB0AFC49AFE9F4963C47BF62AAF24A8AE436BDA14FD8B"
+        git_url:             "https://oejenkins.blob.core.windows.net/oejenkins/Git-2.19.1-64-bit.exe"
+        git_hash:            "5E11205840937DD4DFA4A2A7943D08DA7443FAA41D92CCC5DAFBB4F82E724793"
+        seven_zip_url:       "https://oejenkins.blob.core.windows.net/oejenkins/7z1806-x64.msi"
+        seven_zip_hash:      "F00E1588ED54DDF633D8652EB89D0A8F95BD80CCCFC3EED362D81927BEC05AA5"
+        vs_buildtools_2017:  "https://oejenkins.blob.core.windows.net/oejenkins/vs_buildtools_2017.exe"
+        vs_buildtools_hash:  "6F49872B04A0EAEDF5ED96AB25F7697062D81A419D45D9970E41784F31165BF2"
```

Comment:
vs_buildtools_hash I think it is a typo here , this parameter is being called as vs_buildtools_2017_hash few lines down. 
It is missing 2017 from the name